README.md

```diff
@@ -63,71 +119,129 @@ The following states are used to represent the FT's service tiers:
 <span class="o-labels o-labels--support-bronze">Bronze</span>
 ```
 
-## Sass
+### Indicator Label Markup
+
+Indicator labels have one of three statuses:
+- `live`
+- `updated`
+- `new`
```

Comment:
For live blogs, we also have a `closed` status which makes it clear that an old live blog is closed. Would it be possible to add this status? Otherwise, we'll need to add a different attribute to the markup and style it separately in `next-article`. We use the `live` and `closed` statuses the most in live blogs.